pyproject.toml

```diff
@@ -18,7 +18,56 @@ write_to = "astropy/_version.py"
     [tool.astropy-bot.changelog_checker]
         enabled = false
 
+[tool.black]
+line-length = 100
+force-exclude = '''
+^/(
+  (
+    # Directories to always exclude (.gitignore is always excluded)
+      \.pyinstaller
+    | docs
+    | examples
+    | astropy/_dev
+    | astropy/_erfa
+    | astropy/config
+    | astropy/constants
+    | astropy/convolution
+    | astropy/coordinates
+    | astropy/cosmology
```

Comment:
include cosmology! Or do you want to do application in followup PRs?

Reply:
I think doing them in followup PRs would be better, unless we end up deciding everything should be blacked. In which case everything should be in one giant change?

Reply:
It might be easier to apply ``black`` globally by doing it sequentially to supbackages.  Merge conflicts abound while waiting for subpackage maintainer approval.

Reply:
Close now that black APE is accepted?